src/Mongo2Go/Helper/FileSystem.cs

```diff
@@ -31,7 +32,13 @@ public void DeleteFile(string fullFileName)
         public void MakeFileExecutable (string path) 
         {
             //when on linux or osx we must set the executeble flag on mongo binarys
-            File.SetAttributes (path, (FileAttributes)((int)File.GetAttributes (path) | 0x80000000));
+            var p = Process.Start("chmod", $"+x {path}");

```

Comment:
Windows?! :-(

Reply:
I did run them on Linux, but I confess I fiddled a bit with the project settings to build it for .NET Core 2.0 instead - since that change did a lot of stuff I wasn't sure you actually liked, I kept it out of this PR.

I'm of course happy to submit a separate PR with these changes as well, it might be that some of them are applicable for you.

Re: `MakeFileExecutable`, I think that code is only executed for Linux as it is (https://github.com/Mongo2Go/Mongo2Go/blob/master/src/Mongo2Go/MongoDbRunner.cs#L135), but to be sure we could have a test and run separate code paths for different OS

Reply:
You are absolutely right. I did a quick lookup and haven't seen `RuntimeInformation.IsOSPlatform(OSPlatform.Linux)`. We might want to rename the method from `MakeFileExecutable` to `MakeFileExecutableForLinux`! 😄

Reply:
>  to build it for .NET Core 2.0 instead
So you wanted to go from `<TargetFramework>netstandard1.6</TargetFramework>` to `<TargetFramework>netcoreapp2.0</TargetFramework>`. Are there any benefits for you to do that?!

Reply:
Not entirely sure what I'm doing, I find the combination of versions in the new MS world very confusing! 😄 

As you can see in #44, I updated the test project from `netcoreapp1.0` to `netcoreapp2.0`, but it might be that some version in between works as well...

Reply:
Yeah. Versions. You will get used to that. It makes sense in the end... 😕 😄 

Having the Tests in `netcoreapp2.0` is no breaking change at all. If they run again, that will be a bug benefit! Great! :-)

Reply:
I have made the chmod change locally (I have not got round to creating a PR) and it is working fine on Linux. I also renamed the method to MakeFileExecutableForLinux!

Reply:
I also added the `chmod` for macOS. Does anybody here has a Mac!?